### PR TITLE
Update nav on the HTML cheat sheets

### PR DESCRIPTION
**Pull request by hectorsector**
The navigation on some cheat sheets that are generated in HTML is outdated. The changes introduced here should update the navigation to more closely resemble the nav used by the training team.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -5,9 +5,9 @@
         <ul class="text-left">
           <li>&copy; {{ site.time | date: '%Y' }} <span>GitHub</span>, Inc.</li>
           <li><a href="https://github.com/site/terms">Terms</a></li>
-          <li><a href="https://training.github.com/kit/license.html">License</a></li>
+          <li><a href="{{ site.baseurl }}/license.html">License</a></li>
```
Comment (crichID) [inline _includes/footer.html]: @hectorsector there doesn't appear to be a `license.html` currently in the `services-web` repo. There is a license.md and a license, but neither of them renders at `services.github.com/license.html`. This is definitely something we should try to fix when we update this submodule!

**Comment by crichID** [comment]
@hectorsector I tested this locally and everything looks :+1:! I say :shipit:

File: _includes/footer.html (hunk 1 above)
Comment (hectorsector) [inline _includes/footer.html]: Thanks for checking! `{{ site.baseurl }}` points to the kit in particular, not services.

**Comment by hollenberry** [comment]
Everything looks 👍 on my end too! Awesome work, @hectorsector ! 🎉

**Pull request merged**